src/pages/AppList/AppListClass.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+// import { Link } from 'react-router-dom';
+import { ListViewIcon, ListViewItem } from 'patternfly-react';
+import { IstioLogo } from '../../config';
+// import { PfColors } from '../../components/Pf/PfColors';
+import { AppList, AppListItem } from '../../types/AppList';
+
```

Comment:
Why are commented imports?

Reply:
A I saw there are commented by the LINK. OK

Reply:
True, I commented when I disabled the link for the details page.
I will clean them, yeah, I forgot.